src/workerd/io/worker-interface.capnp

```diff
@@ -147,14 +147,31 @@ struct QueueMessage @0x944adb18c0352295 {
   contentType @3 :Text;
 }
 
+struct QueueRetryBatch {
+  retry @0 :Bool;
+  union {
+    undefined @1 :Void;
+    delaySeconds @2 :Int32;
+  }
+}
+
+struct QueueRetryMessage {
+  msgId @0 :Text;
+  union {
+    undefined @1 :Void;
+    delaySeconds @2 :Int32;
+  }
+}
+
 struct QueueResponse @0x90e98932c0bfc0de {
   outcome @0 :EventOutcome;
-  retryAll @1 :Bool;
-  ackAll @2 :Bool;
-  explicitRetries @3 :List(Text);
-  # List of Message IDs that were explicitly marked for retry
-  explicitAcks @4 :List(Text);
-  # List of Message IDs that were explicitly marked as acknowledged
+  ackAll @1 :Bool;
+  retryBatch @2 :QueueRetryBatch;
```

Comment:
Feel free to ignore, but the naming difference in ack*All* vs retry*Batch* feels unnecessary vs aligning them both on either "all" or "batch".

Reply:
The naming difference was introduced to deal with the breaking change. Queues code is expecting `retryAll` to be a boolean (runtimes in different versions can send either `retryAll` or `retryBatch` for some period of time, I presume). I know that we can make `retryAll` to accept one of two types, but the resulting code to deal with that would be less elegant than simply maintaining this naming difference. Queues code is already implemented following this approach. I propose we revisit this under MQ-435 (cleanup task to remove deprecated fields from queue code).